apache-rat-core/src/main/java/org/apache/rat/config/exclusion/ExclusionUtils.java

```diff
@@ -209,110 +185,4 @@ private static void verifyFile(final File file) {
             throw new ConfigurationException(format("%s is not a valid file.", file));
         }
     }
-
```

Comment:
@Claudenw I'm a bit suspicious if I removed too much ....
some of the methods are used by Reflection in other PRs and failed the build. But the build runs through .... is that code really unused or am I missing something here?